src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.cpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp"
+
+#include <pup.h>  // IWYU pragma: keep
+
+#include "DataStructures/Tensor/Tensor.hpp"  // IWYU pragma: keep
+#include "Utilities/MakeWithValue.hpp"
+
+namespace Elasticity {
+namespace ConstitutiveRelations {
+
+template <size_t Dim>
+IsotropicHomogeneous<Dim>::IsotropicHomogeneous(double incompressibility,
+                                                double rigidity) noexcept
+    : incompressibility_(incompressibility), rigidity_(rigidity) {}
+
+template <>
+template <typename DataType, typename Frame>
+tnsr::II<DataType, 3, Frame> IsotropicHomogeneous<3>::stress(
+    const tnsr::ii<DataType, 3, Frame>& strain,
+    const tnsr::I<DataType, 3, Frame>& /*x*/) const noexcept {
```

Comment:
I'm confused why you are passing x at all...does this have to do with the elliptic solver somehow?

Reply:
see [above](https://github.com/sxs-collaboration/spectre/pull/1298#discussion_r258469238)